### DISCUSSION TITLE
删去直播间部分老旧无用功能

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#423] [General]
1.建议删去 直播间-榜单，下的 金瓜子榜、今日礼物榜、七日礼物榜。这几个都是废品，被淘汰很久了，留着很多余。

其他建议，随便做不做的，没什么用的功能
1.汇率更改，把钱包余额和礼物价值的 金瓜子 改成 电池。（单纯的看着舒服点）
2.做个直播间高能榜？ （没什么用，只是发帖了就顺便提一下）

**Comment by GD-Slime** [comment]
这几个功能（除了金瓜子改电池）都已经在dev分支最新的几个commit中完成 (#410)。 请等待新版本发布，或者可以自行编译dev代码尝鲜。
